public/data/swaroop-2021.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Swaroop Anegundi",
+  "location": "Hubli, Karnataka, India",
+  "bio": "MERN Stack Devloper and Competitive Programmer",
+  "avatar": "",
```

```suggestion
  "avatar": "",
```
Add avatar link as mentioned in readme file.